src/test/not_activated/flutter_create/extension.test.ts

```diff
@@ -89,6 +89,72 @@ describe("command", () => {
 		assert.equal(json?.template === "module", true);
 	});
 
+	it("Flutter: New Package Project can be invoked and creates trigger file", async () => {
```

Comment:
Same as above, we could pull a function out here to avoid the duplication (the body of this test was modified recently to check the pre-population of the command, so it could change again and one copy would be easier).